lpa-submissions-e2e-tests/cypress/support/common/validateErrorMessage.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types = "Cypress"/>
+import {
+  errorMessage as errorMessageObject,
+  summaryErrorMessage as summaryErrorMessageObject,
+} from '../../support/PageObjects/common-page-objects';
+
+module.exports = (errorMessage, errorMessageObjectId, summaryErrorMessageObjectId) => {
+  summaryErrorMessageObject(summaryErrorMessageObjectId).should('be.visible');
+  errorMessageObject(errorMessageObjectId).should('be.visible');
+  summaryErrorMessageObject(summaryErrorMessageObjectId)
```

Comment:
This is a common pattern (i.e. check for visibility and containing text. Could be a function or cypress command?

Reply:
Pushed back.